packages/cli/code-pushup.config.mock.ts

```diff
@@ -103,3 +96,6 @@ export default {
     },
   ],
 };
+function join(outputDir: string, arg1: string): string {
+  throw new Error('Function not implemented.');
+}
```

Comment:
Should be removed and replaced with `import { join } from 'path'`. I suspect you clicked on the wrong VS Code suggestion :grin: